src/views/RasterPlot/TimeScrollView/TimeScrollView.tsx

```diff
@@ -147,12 +159,63 @@ export const use2dPanelDataToPixelMatrix = (pixelsPerSecond: number, startTimeSe
     }, [pixelsPerSecond, startTimeSec, dataMin, dataMax, userScaleFactor, panelHeight, invertY])
 }
 
+export const useTimespanToPixelMatrix = (timeToPixelMatrix: Matrix) => {
```

Comment:
Unfortunately, the actual consumer needs to get a time-to-pixels transformation matrix and then manually call this hook--we can't actually call another hook from within this class (because of the `useMemo` callback)